### PR TITLE
Minor modifications to transpositions of identifications along equivalences

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
Did you, similarly to me, give up on trying to factor out separate files for transposing identifications along sections and retractions? 😅

**Comment by EgbertRijke** [comment]
No, I did not attempt that. Is there any problem with doing that?

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 1]
```diff
@@ -118,27 +162,18 @@ module _
   {l1 l2 : Level} {A : UU l1} {B : UU l2} (e : A ≃ B)
   where
 
-  htpy-map-eq-transpose-equiv :
+  compute-map-eq-tranpsose-equiv :
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:165]: `transpose`*

**Comment by fredrik-bakke** [comment]
> No, I did not attempt that. Is there any problem with doing that?

Well, no. It's just a larger job than I think it looks at first glance.

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 2]
```diff
@@ -147,84 +182,70 @@ the original identifications.
...
+    ( left-inv (is-retraction-map-inv-equiv e _))
+```
+
+### Transposing concatenated identifications along equivalences
+
+Transposing concatenated identifications into a triangle with a transpose of the
+left factor.
+
+```agda
+  triangle-eq-transpose-equiv-concat :
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:246]: Module is carried over to a new section

**Review by fredrik-bakke — APPROVED** [review]
Looks good apart from my few comments!

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 3]
```diff
@@ -86,16 +103,17 @@ module _
...
 
 ```agda
   eq-transpose-equiv-inv :
     (x : A) (y : B) → (y ＝ map-equiv e x) ≃ (map-inv-equiv e y ＝ x)
   eq-transpose-equiv-inv x y =
-    ( equiv-inv x (map-inv-equiv e y)) ∘e
-    ( eq-transpose-equiv x y) ∘e
-    ( equiv-inv y (map-equiv e x))
+    ( inv-equiv (equiv-ap e _ _)) ∘e
+    ( equiv-concat (is-section-map-inv-equiv e y) _)
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:116]:
```suggestion
    ( inv-equiv (equiv-ap e (map-inv-equiv e y) x)) ∘e
    ( equiv-concat (is-section-map-inv-equiv e y) (map-equiv e x))
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 4]
```diff
@@ -109,7 +127,33 @@ It is sometimes useful to consider identifications `y ＝ e x` instead of
...
+  {l1 l2 : Level} {A : UU l1} {B : UU l2} (e : A ≃ B)
+  where
+
+  compute-refl-eq-transpose-equiv :
+    {x : A} →
+    map-eq-transpose-equiv e refl ＝ inv (is-retraction-map-inv-equiv e x)
+  compute-refl-eq-transpose-equiv =
+    map-eq-transpose-equiv-inv
+      ( equiv-ap e _ (map-inv-equiv e _))
+      ( ap inv (coherence-map-inv-equiv e _) ∙
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:143]:
```suggestion
  compute-refl-eq-transpose-equiv {x} =
    map-eq-transpose-equiv-inv
      ( equiv-ap e x (map-inv-equiv e (map-equiv e x)))
      ( ap inv (coherence-map-inv-equiv e x) ∙
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 5]
```diff
@@ -109,7 +127,33 @@ It is sometimes useful to consider identifications `y ＝ e x` instead of
...
+        inv (ap-inv (map-equiv e) _))
+
+  compute-refl-eq-transpose-equiv-inv :
+    {x : A} →
+    map-eq-transpose-equiv-inv e refl ＝ is-retraction-map-inv-equiv e x
+  compute-refl-eq-transpose-equiv-inv {x} =
+    map-eq-transpose-equiv-inv
+      ( equiv-ap e _ _)
+      ( ( right-unit) ∙
+        ( coherence-map-inv-equiv e _))
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:153]:
```suggestion
      ( equiv-ap e (map-inv-equiv e (map-equiv e x)) x)
      ( ( right-unit) ∙
        ( coherence-map-inv-equiv e x))
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 6]
```diff
@@ -118,27 +162,18 @@ module _
...
   where
 
-  htpy-map-eq-transpose-equiv :
+  compute-map-eq-tranpsose-equiv :
     {x : A} {y : B} →
     map-eq-transpose-equiv e {x} {y} ~ map-eq-transpose-equiv' e
-  htpy-map-eq-transpose-equiv {x} refl =
+  compute-map-eq-tranpsose-equiv {x} refl =
     ( map-eq-transpose-equiv-inv
       ( equiv-ap e x _)
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:170]:
```suggestion
      ( equiv-ap e x (map-inv-equiv e (map-equiv e x)))
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 7]
```diff
@@ -147,84 +182,70 @@ the original identifications.
...
-                      ( map-inv-equiv
-                        ( equiv-ap e x (map-inv-equiv e y))
-                        ( ( inv p) ∙
-                          ( inv (is-section-map-inv-equiv e y))))))
-                  ( is-section-map-inv-equiv e y)) ∙
-                ( triangle-eq-transpose-equiv (inv p))))) ∙
-          ( ap-inv (map-equiv e) (map-eq-transpose-equiv-inv e p))))
+    inv
+      ( is-section-map-inv-equiv
+        ( equiv-ap e _ _)
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:206]:
```suggestion
        ( equiv-ap e (map-inv-equiv e y) x)
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 8]
```diff
@@ -147,84 +182,70 @@ the original identifications.
...
-    inv
-      ( map-inv-equiv
-        ( equiv-left-transpose-eq-concat'
-          ( refl)
-          ( map-eq-transpose-equiv e (inv p))
-          ( ap (map-inv-equiv e) p ∙ is-retraction-map-inv-equiv e x))
-        ( right-unit ∙ triangle-eq-transpose-equiv-inv' p))
+  right-inverse-eq-transpose-equiv {x} refl =
+    ( right-whisker-concat (compute-refl-eq-transpose-equiv e) _) ∙
+    ( left-inv (is-retraction-map-inv-equiv e _))
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:237]:
```suggestion
    ( right-whisker-concat
      ( compute-refl-eq-transpose-equiv e)
      ( is-retraction-map-inv-equiv e x)) ∙
    ( left-inv (is-retraction-map-inv-equiv e x))
```

File: src/foundation/transposition-identifications-along-equivalences.lagda.md [hunk 9]
```diff
@@ -147,84 +182,70 @@ the original identifications.
 ```agda
   triangle-eq-transpose-equiv :
```
Comment (fredrik-bakke) [inline src/foundation/transposition-identifications-along-equivalences.lagda.md:183]: Module is carried over to new section here too

Comment (fredrik-bakke) [reply src/foundation/transposition-identifications-along-equivalences.lagda.md:183]: (Due to your introduction of a new section header)

**Comment by EgbertRijke** [comment]
I addressed the comments, except the ones that were about filling in `_` because I actually like this use of underscore and I am much more comfortable with this use of `_` than with the `_` in partially evaluated mixfix operators.

**Pull request merged**